packages/video_player/elinux/gst_video_player.cc

```diff
@@ -403,6 +403,8 @@ void GstVideoPlayer::GetVideoSize(int32_t& width, int32_t& height) {
     return;
   }
 #endif  // USE_EGL_IMAGE_DMABUF
+
+  gst_caps_unref(caps);
```

Comment:
Please make another pull request if the content is different from next time.

Reply:
I'm Sorry.
It was already a merged patch...